ios/Framework/Core/DDL/FormScreenlet/Interactors/DDLFormLoadFormInteractor.swift

```diff
@@ -66,7 +65,7 @@ class DDLFormLoadFormInteractor: ServerReadConnectorInteractor {
 					key: "structureId-\(loadCon.structureId)") { record, attributes in
 
 				loadCon.resultRecord = record as? DDLRecord
-				loadCon.resultUserId = attributes?["userId"]?.int64Value
+				loadCon.resultUserId = attributes?["userId"] as? Int64
 
```

Comment:
If the server is returning the userId as a string this will crash

Reply:
Yes, in these cases, we should convert the object as AnyObject and then get the `int64Value` and avoid future crashes. Thanks Victor!